README.md

```diff
@@ -1 +1,162 @@
-# blog-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+</div>
+
+
+> # Blog App
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)  
+- [🚀 Video Presentation ](#video-project-presentation)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Catalog of my things] <a name="about-project"></a>
+
+**[Catalog of my things]** Blog app is a fully funcioning website which helps people post blogs and add comments, add likes, and create account and login. 
```

Comment:
- [ ] Kindly update the name of the project in the `about-project` section to reflect the current project.